compute_tools/src/bin/compute_ctl.rs

```diff
@@ -149,6 +149,8 @@ struct Cli {
 fn main() -> Result<()> {
     let cli = Cli::parse();
 
+    let scenario = failpoint_support::init();
+
     // For historical reasons, the main thread that processes the spec and launches postgres
```

Comment:
Why is this moved here? No objections, seems fine to me, but I wonder why edition 2024 needs that.

Reply:
The idea was to run it before any threads executed before it. The function internally does a `std::env::remove_var("FAILPOINTS");`, which with the 2024 edition becomes unsafe. The underlying functionality is already dangerous and can cause segfaults, but I used the opportunity of the 2024 switch to audit that location for whether there is threads running already.

Some further reading:

https://doc.rust-lang.org/nightly/std/env/fn.remove_var.html#safety
https://github.com/rust-lang/rust/pull/24741
https://github.com/chronotope/chrono/issues/499

Rust's standard library uses locks to ameliorate things a little bit, but those locks are not used by C code that modifies or reads the environment, sometimes even C code that comes from the standard library. So they made it unsafe.